R/language_to_ms_name.R

```diff
@@ -21,11 +21,15 @@ ms_language_to_ms_name = function(
   return(xname)
 }
 
-#' @rdname ms_language_to_ms_name
+
+#' Validate whether a specified language and gender combination is supported
+#'
+#' @param language
```

Comment:
Please define these parameters in the documentation - these are missing currently.